spec/models/email_link_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.describe EmailLink, type: :model do
+    it "default factory is valid" do
+       expect(build(:email_link).valid?).to be_truthy
+
+
+    end
+
+    it "fails validations with invalid attributes" do
+        expect(build(:email_link, url:"").valid?).to be_falsy
+    end
+
+    it "returns undecided emails" do
+
+        email1 = build(:email_link)
+        email2 = build(:email_link, accept_or_rejected_at: DateTime.now, accepted: true)
```

Comment:
That should have worked. Can you push the code you tried to a new branch and open a pull request (to merge into this branch) and I can take a look?